dnscrypt-proxy/config.go

```diff
@@ -396,9 +396,9 @@ func ConfigLoad(proxy *Proxy, flags *ConfigFlags) error {
 				return fmt.Errorf("Bootstrap resolver [%v]: %v", resolver, err)
 			}
 		}
-		proxy.xTransport.ignoreSystemDNS = config.IgnoreSystemDNS
```

Comment:
If there are no bootstrap resolvers (?), using the system resolvers feels reasonable.